qemu/tests/test_control_virtqueue.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_net
+from virttest.utils_test.qemu import MemoryHotplugTest
+
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+class MemoryHotplugRepeat(MemoryHotplugTest):
+
+    def hotplug_mem(self, vm, target_mem):
+        """
+        Hotplug memory in target_mem
+        :param vm: vm object in this test
+        :param target_mem: target memory to be hotplugged
+        """
+        self.hotplug_memory(vm, target_mem)
+        self.check_memory(vm)
+
+    def start_test(self):
+        """
+        Prepare required test params, hotplug 1 memory to update device
+        """
+        vm = self.env.get_vm(self.params["main_vm"])
+        session_serial = vm.wait_for_serial_login()
+        target_mem = self.params["target_mem"]
+        error_context.context("Hotplug memory %s" % target_mem, LOG_JOB.info)
+        self.hotplug_memory(vm, target_mem)
```

Comment:
You redefine a method `hotplug_mem` but seem not to use it.

Reply:
And I think the new class does not have any repeat operations but it named `MemoryHotplugRepeat`

Reply:
I messed up this steps while automating it, I will replace it with a simpler way in the next update.